doc/source/whatsnew/v0.18.1.rst

```diff
@@ -185,10 +185,11 @@ Partial string indexing now matches on ``DateTimeIndex`` when part of a ``MultiI
 
    dft2 = pd.DataFrame(np.random.randn(20, 1),
                        columns=['A'],
-                       index=pd.MultiIndex.from_product([pd.date_range('20130101',
-                                                                       periods=10,
-                                                                       freq='12H'),
-                                                        ['a', 'b']]))
+                       index=pd.MultiIndex.
+                       from_product([pd.date_range('20130101',
```

Comment:
I found this indentation confusing. If the lines are too long, can we use something like:
```
dft2 = pd.DataFrame(
    np.random.randn(20, 1),
    ...
```

Reply:
Thanks for the comment. I wasn't flexible with that. I'll fix it.